example/lib/pages/posts_list/posts_list_page_scenes.dart

```diff
@@ -52,13 +53,35 @@ class EmptyListScene extends BasePostsListScene {
 
 /// A Scene showing the [PostsListPage] with [Post]s.
 class WithPostsScene extends BasePostsListScene {
+  int _numPosts = Post.fakePosts().length;
+
   @override
   String get title => 'With Posts';
 
+  @override
+  List<WidgetBuilder> get environmentControlBuilders => <WidgetBuilder>[
+        (BuildContext context) {
+          return NumberStepperControl(
+            title: const Text('# Posts'),
+            value: _numPosts,
+            onDecrementPressed: () async {
+              _numPosts = max(0, _numPosts - 1);
+              rebuildScene();
+            },
+            onIncrementPressed: () async {
+              _numPosts = min(_numPosts + 1, Post.fakePosts().length);
```

Comment:
Wow, it really doesn't add that much boilerplate to just implement this stuff "manually" in the app

Reply:
Yeah! I had initially shied away from this, thinking that it would be easier for users to declare a more abstract config option, but this is about the same amount of code and offers ~infinite configurability. Glad you like it! :)